packages/react/src/patterns/sub-patterns/Layout/Layout.js

```diff
@@ -22,29 +22,34 @@ const { prefix } = settings;
  */
 const _types = {
   '1-3': [`${prefix}--col-lg-4`, `${prefix}--col-lg-12`], // 1/4 - 3/4
+  '2-3': [`${prefix}--col-lg-8`, `${prefix}--col-lg-4`], // 2/3 - 1/3
```

Comment:
I don't think this is going to align with the grid properly. These classes are based on a 16 column grid, and you can't divide that into 3.

Reply:
I've implemented a new proportion based styling